x-pack/plugins/apm/public/components/app/TraceOverview/TraceList.tsx

```diff
@@ -34,19 +33,19 @@ const traceListColumns: ITableColumn[] = [
     name: 'Name',
     width: '40%',
     sortable: true,
-    render: (_, group: ITransactionGroup) => (
-      <TooltipOverlay content={group.sample.transaction.name}>
+    render: (name, group: ITransactionGroup) => (
+      <TooltipOverlay content={name}>
         <StyledTransactionLink transaction={group.sample}>
-          {group.sample.transaction.name}
+          {name}
         </StyledTransactionLink>
       </TooltipOverlay>
     )
   },
   {
-    field: 'sample',
+    field: 'sample.context.service.name',
     name: 'Originating service',
     sortable: true,
-    render: (transaction: Transaction) => transaction.context.service.name
+    render: service => service
```

Comment:
I think this can be removed as far as EUI is concerned (and it defaults to something like identity) but you probably get an error if you remove it due to our shadowed ITableColumn type interface that isn't from EUI :( If you make `render` optional in the `ITableColumn` type, you can remove it here. Eventually EUI will be in TS and we won't have this kind of problem...